config/taxonomy/Makefile

```diff
@@ -1,27 +1,32 @@
 ROOT_DIR := ../..
 include $(ROOT_DIR)/Makefile.env
 include $(ROOT_DIR)/hack/make-rules/tools.mk
-
 .DEFAULT_GOAL := all
-
-all: clean compile-taxonomy-schema validate
-
-compile-taxonomy-schema:
-	go run $(ROOT_DIR)/main.go taxonomy compile --base ./base/base.yaml --out ./objects/taxonomy.json \
-		--codegen
-
-validate:
+all: clean objects/taxonomy.json generate-base generate-client
+objects/taxonomy.json:
+	go run $(ROOT_DIR)/main.go taxonomy compile --base ./base/base.yaml --out ./objects/taxonomy.json
 	go run $(ROOT_DIR)/main.go taxonomy validate ./objects/policy_manager_request.json
 	go run $(ROOT_DIR)/main.go taxonomy validate ./objects/policy_manager_response.json
-
-compile-codegen-cleanup: clean $(TOOLBIN)/openapi-generator-cli compile-taxonomy-schema
+.PHONY: generate-base
+generate-base: $(TOOLBIN)/openapi-generator-cli objects/taxonomy.json
 	mkdir -p $(ROOT_DIR)/pkg/taxonomy/model/base
 	$(TOOLBIN)/openapi-generator-cli generate -i ./codegen.spec.yaml -g go -o $(ROOT_DIR)/pkg/taxonomy/model/base --global-property=models,modelDocs=false \
-	--additional-properties=packageName=base \
-	--enable-post-process-file \
-	--type-mappings=object=interface{}
-	rm -f objects/taxonomy.json
-
+		--additional-properties=packageName=base \
+		--enable-post-process-file \
+		--type-mappings=object=interface{}
+.PHONY: generate-client
+generate-client: $(TOOLBIN)/openapi-generator-cli objects/taxonomy.json
+	mkdir -p $(ROOT_DIR)/pkg/connectors/openapiclient
+	$(TOOLBIN)/openapi-generator-cli generate -g go --additional-properties=packageName=openapiclient,isGoSubmodule=false \
+		--global-property=apis,supportingFiles -o $(ROOT_DIR)/pkg/connectors/openapiclient \
```

Comment:
can modify `--global-property=apis,supportingFiles `to `--global-property=apis,supportingFiles,apiDocs=false ` since we dont need the api docs to be generated.

Reply:
Done